packages/xo-server/src/xo-mixins/file-restore-ng.js

```diff
@@ -190,13 +197,50 @@ export default class BackupNgFileRestore {
 
   @defer
   async listBackupNgDiskPartitions($defer, remoteId, diskId) {
+    const app = this._app
+    const { proxy, url, options } = await app.getRemoteWithCredentials(remoteId)
+    if (proxy !== undefined) {
+      const stream = await app.callProxyMethod(
+        proxy,
+        'backup.listDiskPartitions',
+        {
+          disk: diskId,
+          remote: {
+            url,
+            options,
+          },
+        },
+        { expectStream: true }
```

Comment:
IMHO, we should handle this better due to the integration of `binary` responses.